shell/platform/darwin/macos/BUILD.gn

```diff
@@ -109,6 +109,8 @@ executable("flutter_desktop_darwin_unittests") {
 
   cflags_objcc = [ "-fobjc-arc" ]
 
+  ldflags = [ "-ObjC" ]
```

Comment:
I thought this was implicit. The default rules should have already been building this target and would have failed linking without this flag.

Reply:
Not sure why it was needed, i'm going to leave it here since it shouldn't create additional problems.